From 08a1f7b97b99d6508281b282bd5f52f21ada2575 Mon Sep 17 00:00:00 2001
From: Chris Wilson <chris@chris-wilson.co.uk>
Date: Wed, 5 Oct 2016 18:40:56 +0100
Subject: [PATCH 4323/4706] drm: Fix up kerneldoc for new
 drm_gem_dmabuf_export()

commit 67c8f116f5d59df46c2be99674542b43195b21d7 upstream

I hit send before completing a make htmldoc, and lo I forgot to fix up
the cut'n'paste.

Fixes: a4fce9cb782a ("drm/prime: Take a ref on the drm_dev when exporting...")
Reported-by: kbuild test robot <fengguang.wu@intel.com>
Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
Cc: Daniel Vetter <daniel.vetter@ffwll.ch>
Cc: stable@vger.kernel.org
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Link: http://patchwork.freedesktop.org/patch/msgid/20161005174056.29869-1-chris@chris-wilson.co.uk
---
 drivers/gpu/drm/drm_prime.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/drm_prime.c b/drivers/gpu/drm/drm_prime.c
index cc70a44..4bae994 100644
--- a/drivers/gpu/drm/drm_prime.c
+++ b/drivers/gpu/drm/drm_prime.c
@@ -285,7 +285,8 @@ static void drm_gem_unmap_dma_buf(struct dma_buf_attachment *attach,
 
 /**
  * drm_gem_dmabuf_export - dma_buf export implementation for GEM
- * @dma_buf: buffer to be exported
+ * @dev: parent device for the exported dmabuf
+ * @exp_info: the export information used by dma_buf_export()
  *
  * This wraps dma_buf_export() for use by generic GEM drivers that are using
  * drm_gem_dmabuf_release(). In addition to calling dma_buf_export(), we take
-- 
1.7.5.4

